WalletWasabi.Gui/Controls/WalletExplorer/WalletActionViewModel.cs

```diff
@@ -1,28 +1,12 @@
-using Avalonia.Threading;
-using AvalonStudio.Extensibility;
-using AvalonStudio.Shell;
-using ReactiveUI;
-using System.Reactive;
-using System.Threading.Tasks;
-using WalletWasabi.Blockchain.Keys;
 using WalletWasabi.Gui.ViewModels;
-using WalletWasabi.Services;
 
 namespace WalletWasabi.Gui.Controls.WalletExplorer
 {
 	public class WalletActionViewModel : WasabiDocumentTabViewModel
 	{
-		public WalletViewModel Wallet { get; }
-
-		public WalletService WalletService => Wallet.WalletService;
-		public KeyManager KeyManager => WalletService.KeyManager;
-		public bool IsWatchOnly => KeyManager.IsWatchOnly;
-		public bool IsHardwareWallet => KeyManager.IsHardwareWallet;
-
-		public WalletActionViewModel(string title, WalletViewModel walletViewModel)
+		public WalletActionViewModel(string title)
```

Comment:
How about put `private WalletService WalletService { get; }` here? This is called WalletActionViewModel so it is Wallet dependent by name.

Reply:
Added a commit to remove this class, its redundant, Im fairly sure its a left over from a hack.

Getting rid of it makes more sense.